File: Pemrograman1/Modul5/C/PRAK501-2310817310009-MuhammadFauzanAhsani.c
#include <stdio.h>

int MaxBilangan(int a, int b, int c, int d){
    if (a > b && a > c && a > d){
        return a;}
    else if (b > a && b > c && b > d){
        return b;}
    else if (c > a && c > b && c > d){
        return c;}
    else if (d > a && d > b && d > c){
        return d;}
}

int main(){
    int a, b, c, d;
    scanf("%d %d %d %d", &a, &b, &c, &d);
    int hasil = MaxBilangan(a, b, c, d);
    printf("%d", hasil); 
    return 0;
}

File: Pemrograman1/Modul5/Python/PRAK501-2310817310009-MuhammadFauzanAhsani.py
def MaxBilangan(a:int, b:int, c:int, d:int):
    if (a > b and a > c and a > d):
        return a
    elif (b > a and b > c and b > d):
        return b
    elif (c > a and c > b and c > d):
        return c
    elif (d > a and d > b and d > c):
        return d
    
a, b, c, d = map(int, input().split())
hasil = MaxBilangan(a, b, c, d)
print(hasil)